modules/tinymce/src/plugins/help/main/ts/ui/PluginsTab.ts

```diff
@@ -47,8 +80,13 @@ const tab = (editor: Editor): Dialog.TabSpec & { name: string } => {
 
   const pluginLister = (editor: Editor) => {
     const pluginKeys = getPluginKeys(editor);
-    const pluginLis = Arr.map(pluginKeys, (key) => {
-      return '<li>' + maybeUrlize(editor, key) + '</li>';
+    const sortedPluginData = Arr.sort(
```

Comment:
This will need a changelog entry won't it as it is a visible change for everyone?

Reply:
I crossed out changelog because the other change (autocorrect) didn't have an entry, so I just assumed that was how these changes work.

Reply:
But should it have had a changelog entry? Or just because things are now sorted?

Reply:
The premium names generally don't have a changelog since it's not something opensource/non-premium customers would be impacted by. However the sorting impacts everyone, so it should have a changelog.

Reply:
I'll add it for https://ephocks.atlassian.net/browse/TINY-9019 then, as that's probably the ticket I ended up doing here.